lib/ansible/modules/cloud/amazon/iam.py

```diff
@@ -451,29 +451,32 @@ def create_group(module=None, iam=None, name=None, path=None):
 
 def delete_group(module=None, iam=None, name=None):
     changed = False
+    del_meta = ''
```

Comment:
I see you added del_meta to this function to be more similar to delete_user(), but since it is never used anywhere (delete_user() doesn't seem to do anything with it either), I feel like we shouldn't add it. It will also make your diff small and tidy. :)